app/create_file.py

```diff
@@ -1 +1,55 @@
-# write your code here
+import sys
+import os
+from datetime import datetime
+
+
+def create_file(path: str, file_name: str) -> None:
+    file_path = os.path.join(path, file_name)
+
+    if not os.path.exists(path):
+        os.makedirs(path)
+
+    mode = "a" if os.path.exists(path) else "w"
```

Comment:
You can use "a" always
"a" - Append - Opens a file for appending, creates the file if it does not exist
```suggestion
    mode = "a"
```